package/tests/testthat/_snaps/cml01/markdown-snaps.md

```diff
@@ -4,18 +4,18 @@
       print(data_snap[[i]])
     Output
       # A tibble: 3,685 x 14
-         ID      AGSXRC TRT01A CMDECOD TRTSD CMASTD ASTDY ADURN CMSTRFL CMENRFL CMDOSE
-         <lstng> <lstn> <lstn> <lstng> <chr> <chr>  <int> <int> <chr>   <chr>    <int>
-       1 BRA-1/~ 47/M/~ A: Dr~ mednam~ 04NO~ 21SEP~   321  -148 No      No          30
-       2 BRA-1/~ 47/M/~ A: Dr~ mednam~ 04NO~ 02DEC~   393   -42 No      No          41
-       3 BRA-1/~ 47/M/~ A: Dr~ mednam~ 04NO~ 15APR~   162  -172 No      No          25
-       4 BRA-1/~ 47/M/~ A: Dr~ mednam~ 04NO~ 15APR~   162  -172 No      No          25
-       5 BRA-1/~ 47/M/~ A: Dr~ mednam~ 04NO~ 15APR~   162  -172 No      No          25
-       6 BRA-1/~ 47/M/~ A: Dr~ mednam~ 04NO~ 19MAY~   196  -165 No      No          22
-       7 BRA-1/~ 47/M/~ A: Dr~ mednam~ 04NO~ 19MAY~   196  -165 No      No          22
-       8 BRA-1/~ 35/F/~ C: Co~ mednam~ 25JU~ 03AUG~   739   -35 No      No          84
-       9 BRA-1/~ 35/F/~ C: Co~ mednam~ 25JU~ 10JAN~   899  -138 No      No           1
-      10 BRA-1/~ 35/F/~ C: Co~ mednam~ 25JU~ 03MAR~   951   -65 No      No          70
+         TRT01A   ID    AGSXRC CMDECOD TRTSD CMASTD ASTDY ADURN CMSTRFL CMENRFL CMDOSE
+         <lstng_> <lst> <lstn> <lstng> <chr> <chr>  <int> <int> <chr>   <chr>    <int>
```

Comment:
should this be lstng_ky? but we have two types lstng and lstng_?

Reply:
Hi @shajoezhu,

That is very strange. As far as I can tell, they're all `lstng_ky` type columns, but the type label is shortened in the snapshot depending on the contents/label of the column.

Reply:
is there a print method for tibble and specify the column width?

Reply:
hi @edelarua , using the tibble print width option will be able to help us to get more span, i think a lot of these info were just truncated. i m less worried about this now. but it would be nice to capture these, I think, and have the full word "lstng_ky‘ spelt 

e.g. `print(as.tibble(lsting), width = 150)`

Reply:
@shajoezhu all listing columns are now fully expanded in test snapshots